test/shared/helpers/serverToClientJson.test.js

```diff
@@ -0,0 +1,28 @@
+var Handlebars = require('handlebars').create(),
+    sinon = require('sinon'),
+    chai = require('chai'),
+    expect = chai.expect,
+    subject = require('../../../shared/helpers/serverToClientJson')(Handlebars);
+
+describe('serverToClientJson', function () {
+  var data = { t: 'test' };
+
+
+  it('should escape and stringify the object', function () {
```

Comment:
I think a stronger, less flaky test, that doesn't require spying on global functions, would be:

```js
var input = { key: 'права' }

var expectedOutput = '%7B%22key%22%3A%22%u043F%u0440%u0430%u0432%u0430%22%7D'

var actualOutput = subject(input)

expect(actualOutput).to.equal(expectedOutput)
```